src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -211,7 +212,10 @@ class SourcesTree extends Component<Props, State> {
 
     const source = this.getSource(item);
     return (
-      <img className={classnames(getSourceClassnames(source), "source-icon")} />
+      <SourceIcon
+        source={source}
+        shouldHide={icon => ["file", "javascript"].includes(icon)}
```

Comment:
Ooops, sorry I wasn't specific enough.  We *do* want the file and JS icons in the tree -- simply not the tabs.  Remove the `shouldHide={icon => ["file", "javascript"].includes(icon)}` here and this is good to go.  Thank you!

Reply:
oke wait...